docs/cli-ref.rst

```diff
@@ -35,3 +35,46 @@ development and testing.
    :module: airflow.cli.cli_parser
    :func: get_parser
    :prog: airflow
+
+Environment Variables
+'''''''''''''''''''''
+
+.. envvar:: AIRFLOW__{SECTION}__{KEY}
+
+  Any specific key in a section in Airflow.
+  The user may replace the ``{SECTION}`` placeholder with any section
+  and the ``{KEY}`` placeholder with any key in that specified section.
+  For example, the user may initiate "AIRFLOW__CORE__DAGS_FOLDER",
+  where CORE is the section and DAGS_FOLDER is the key within the section.
```

Comment:
```suggestion
  For example, If you want to set the ``dags_folder`` options in ``[core]`` section, then you 
   should set the ``AIRFLOW__CORE__DAGS_FOLDER``  environment variable.
```
There are a few problems here. First, not the best tone of speech. The word may in context suggests that there is another variable name as well. Second, all sentences should address the reader directly. Third,  if you refer to a section name it should always be written in square brackets as a code literal.

Reply:
I hope you do not offend my comments, but I do try to make the documentation accessible to a wide audience.
I believe that every developer should read Google's guide, which highlights some good practices.
https://developers.google.com/style